explore_utils.py

```diff
@@ -56,3 +56,25 @@ def find_most_common_traffic_sources(dataset,num=5):
 
     """
     return dataset.train['trafficSource.source'].value_counts().head(num)
+
+def social_active_customers_revenue(dataset):
+
+    """Find the revenue generated by socially engaged users, determining
+    whether or not they generate more revenue
+
+    args:
+    dataset (Dataset): the google analytics dataset
+
+    returns:
+    the revenue generated by social engaged users, and compares it to
```

Comment:
Looks like you are only returning the mean value of socially engaged users, not normal users. You might want to fix the documentation or change what you return.